data/json/monsters/mammal.json

```
@@ -1895,6 +1895,7 @@
     "weight": "550 kg",
     "hp": 90,
     "speed": 300,
+    "attack_cost": 225,
```

```suggestion
    "attack_cost": 230,
```
I'm making this slight change to see if that allows the tests to fire.